### PR TITLE
my first test pull request

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

**Comment by stevenndungu** [comment]
this is not to be merged. Learning how git works

File: instat/dlgAppend.Designer.vb [hunk 1]
```diff
@@ -121,11 +129,50 @@ Partial Class dlgAppend
         Me.ucrInputNewDataframeName.Size = New System.Drawing.Size(235, 21)
         Me.ucrInputNewDataframeName.TabIndex = 8
         '
+        'TextBox1
+        '
+        Me.TextBox1.Location = New System.Drawing.Point(7, 79)
+        Me.TextBox1.Name = "TextBox1"
+        Me.TextBox1.Size = New System.Drawing.Size(100, 20)
+        Me.TextBox1.TabIndex = 9
```
Comment (dannyparsons) [inline instat/dlgAppend.Designer.vb]: @stevenndungu Great to see you are learning how to use GitHub and make pull requests, well done.
@maxwellfundi I know this is only a practise pull request but you should be getting Steven into the habit of always renaming any new controls with the correct naming conventions.

Comment (stevenndungu) [inline instat/dlgAppend.Designer.vb]: Hi Danny,

Its time I have been learning on how to create new dialogues and also
naming conventions.

Thanks. Happy to hear from you

**Comment by maxwellfundi** [comment]
@stevenndungu this can now be closed

**Pull request closed without merging**